### DISCUSSION TITLE
[Kavita] Report Issue function

### DISCUSSION THREAD

**Discussion by thezak48** [Kareadita/Kavita#2690] [Ideas]
### Idea Description

Ability for to report an issue with series/chapter so that it can be flagged that the Admin role can see it needs to be looked at and fixed. 

Perhaps have some pre-set options like:
- Issue with Chapter <x>
- Issue with Metadata
- Issue with Age Rating
- Issue with Other (Followed by Text Box)

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by DieselTech** [comment]
Something similar to how plex handles this. 

![image](https://github.com/Kareadita/Kavita/assets/30128380/16b13e53-0a10-4e13-a9f7-d6bce410e954)


The only problem I see here is how do we alert the admin in a way that is intuitive. Since currently there isn't a way to grab the admins attention built in. We don't want them to have to stumble upon the media issue page in order to find reports.

**Comment by thezak48** [comment]
Could add on the ability to send emails for "Reported issue for xxx has been fixed, etc"
